packages/openstef-meta/src/openstef_meta/transforms/selector.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2025 Contributors to the OpenSTEF project <short.term.energy.forecasts@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""Transform for dropping rows containing NaN values.
+
+This module provides functionality to drop rows containing NaN values in selected
+columns, useful for data cleaning and ensuring complete cases for model training.
+"""
+
+from typing import override
+
+from pydantic import Field
+
+from openstef_core.base_model import BaseConfig
+from openstef_core.datasets import TimeSeriesDataset
+from openstef_core.datasets.validated_datasets import ForecastInputDataset
+from openstef_core.transforms import TimeSeriesTransform
+from openstef_models.utils.feature_selection import FeatureSelection
+
+
+class Selector(BaseConfig, TimeSeriesTransform):
```

Comment:
Possibly a quick win to add this one already in openstef 4 release. (and put it in openstef-models transforms or openstef-core transforms). Makes this pr smalled.